crud/common/utils.lua

```diff
@@ -52,40 +52,71 @@ function utils.get_space_format(space_name, replicasets)
     return space_format
 end
 
-local system_fields = { bucket_id = true }
+local function put(lines, s, ...)
+    table.insert(lines, string.format(s, ...))
+end
+
+local cache = setmetatable({}, {__mode = 'k'})
 
 function utils.flatten(object, space_format, bucket_id)
-    if object == nil then return nil end
+    local fun = cache[space_format]
```

Comment:
`cache` -> `flatten_functions_cache`?

Reply:
`fun` is used for `luafun`, I think it's better to use `flatten_func` name.

Reply:
> cache -> flatten_functions_cache?

ok

> fun is used for luafun, I think it's better to use flatten_func name.

ok